server/http.go

```diff
@@ -151,3 +158,18 @@ func (p *Plugin) respondWithTemplate(w http.ResponseWriter, r *http.Request, con
 	}
 	return http.StatusOK, nil
 }
+
+func (p *Plugin) respondSpecialTemplate(w http.ResponseWriter, key string, status int, contentType string, values interface{}) (int, error) {
```

Comment:
Nit:  if `key` is always a file path to a template, perhaps rename `key` to something representative of a file path?